spec/system/posts_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe "Posts", type: :system do
+  # 投稿作成
+  let(:post) { Post.create(content: "This is a test post.") }
+
+  before do
+    driven_by(:rack_test)
+  end
+  # ポストが作成されることを確認する
+  it "creates a post" do
+    visit new_post_path
+    fill_in "コンテンツ", with: "This is a test post."
+    click_button "保存"
```

Comment:
保存という言葉に違和感がありました。
あと、click_on をつかうと、クリックする要素がリンクでもボタンでも押せるので変更に強いです。
```suggestion
    click_on "投稿する"
```